start.py

```diff
@@ -0,0 +1,62 @@
+import GoogleApi
+import drive
+import sys
+import gmail
+
+
+# ******************************************************
+# *********** PUT CONFIGURATION FILE ID HERE ***********
+fileID = "18Hzz__cX7K3fv_tpZmJsa620EXYwPj0d"
+fileName = sys.argv[1]
+# ******************************************************
+# ******************************************************
+
+
+def yamlID():
+    return fileID
+
+
+def yamlName():
+    return fileName
+
+
+def main():
+    # print("Number of arguments: ", len(sys.argv))
+    # print("The arguments are: ", str(sys.argv))
+    # answer = input("Would you like to send the report by email? (Y/N): ")
+    if (sys.argv[1].find(".yml") == -1):
```

Comment:
Kao što rekoh, ovo puca ako pozovem program bez argumenata. 

Nemoj ovako da parsiraš argumente, nepotrebno komplikovano, skupo i nadasve klimav kod. 

Koristi `argparser`: https://docs.python.org/2/library/argparse.html

Reply:
Trenutno radi, ako stignem veceras cu da implementiram sa argparserom.